ansible/roles/test/tasks/everflow_testbed/everflow_main.yml

```diff
@@ -0,0 +1,37 @@
+- fail:
+    msg: "Mirror stage is not defined or invalid"
+  when: mirror_stage is not defined or mirror_stage not in ['ingress', 'egress']
+
+- fail:
+    msg: "ACL stage is not defined or invalid"
+  when: acl_stage is not defined or acl_stage not in ['ingress', 'egress']
+
+- name: Set flag that recover from config_db.json is needed (default - false)
+  set_fact:
+    recover_from_cfgdb_file: False
+
+- name: Apply Everflow configuration.
+  include: "roles/test/tasks/everflow_testbed/apply_config.yml"
+  tags: everflow_tb_configure
+
+- name: Run Everflow tests.
```

Comment:
suggest to change to 'Run Everflow tests [tor]' as there is also another test for spine. special word to differentiate between the runs for debug